src/app/components/Header/MinimizationTriangle.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <button
+    :aria-label="label"
+    class="minimization-triangle"
+    style="float: right;"
+    :class="{'flipped': value && !settings,
+    'settings-flipped': value && settings,
+    'settings':settings}"
+    @click="$emit('input', !value)"
+  >
+    <v-style>
+      button:focus {
+        border: {{$vxm.chat.tabbing ? '' : 'none !important'}};
+        outline: {{$vxm.chat.tabbing ? '' : 'none !important'}};
```

Comment:
In hindsight, it would have been a much better idea to use a class. That's fixed.
> Just to make sure - this makes it so the triangle button doesn't get the blue outline if the user clicks on it, only if it gets tabbed to, right?

Yes, this ensures focus rings are only visible when the tab key has been pressed.